stable/mongodb/values.yaml

```diff
@@ -144,6 +143,11 @@ podAnnotations: {}
 # Additional pod labels to apply
 podLabels: {}
 
+## Use an alternate scheduler, e.g. "stork".
+## ref: https://kubernetes.io/docs/tasks/administer-cluster/configure-multiple-schedulers/
+##
+# schedulerName:
```

Comment:
Apply the same changes in the `values-production.yaml` too

Reply:
Thanks for catching it. Update done